app/services/calculators/multiples/same_proceedings.rb

```diff
@@ -15,6 +15,12 @@ def spent_date
         expiry_dates.max
       end
 
+      def spent_date_without_relevant_orders
```

Comment:
I think is worth moving this method to the `BaseMultiplesCalculator` and removing their individual implementations in the subclasses, because it will work for both, same proceedings, and separate proceedings:

- In same proceedings is what we have now so will continue working.
- In separate proceedings, when there is only 1 conviction, and it is a relevant order, the collection will be empty and a `max` will return `nil`.